CHANGELOG.md

```diff
@@ -45,6 +56,9 @@ play/pause/rewind/fast-forward/restart/next/previous"
 ## Note on releases below
 The CHANGELOG entries for the releases listed below this were added retroactively and may be incomplete.
 
+## [2.1.2] 2018-10-30
+Version-bump only for *Fire TV Stick 4k*
```

Comment:
nit: I've been italicizing any content that doesn't fit into this pattern so I'd italicize this whole line.